mu-plugins/blocks/horizontal-slider/src/block.js

```diff
@@ -83,7 +83,9 @@ function Block( { items, title } ) {
 		outerRef.current.addEventListener( 'scroll', handleScrollEvent );
 
 		return () => {
-			outerRef.current.removeEventListener( 'scroll', handleScrollEvent );
+			if ( outerRef.current ) {
+				outerRef.current.removeEventListener( 'scroll', handleScrollEvent );
+			}
```

Comment:
[Optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) is a nice way to handle this, looks like [browser support](https://caniuse.com/?search=optional%20chaining) is good

```suggestion
			outerRef.current?.removeEventListener( 'scroll', handleScrollEvent );
```

Reply:
Yep, maybe it's time to move on to optional chaining :) Feels weird.